ILRepack/ILRepack.csproj

```diff
@@ -10,9 +10,9 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>ILRepacking</RootNamespace>
     <AssemblyName>ILRepack</AssemblyName>
-    <TargetFrameworkVersion>v3.5</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
```

Comment:
Ncrunch doesn't build the project because Cecil was 4.0. I can remove it if you want. Don't we want to update it? Theoretically this can be 5.0 and still be able to merge assemblies for v1, v2 or v4.